### ISSUE TITLE
Submitting form containing file data via TDI with jQuery v1.10.1 and IE11 fails

### ISSUE THREAD

**Issue by msevcenko** [twinstone/tdi#8]
Fails with error in Sizzle.setDocument in the following code:

```javascript
	// Support: IE>8
	// If iframe document is assigned to "document" variable and if iframe has been reloaded,
	// IE will throw "permission denied" error when accessing "document" variable, see jQuery #13936
	if ( parent && parent.frameElement ) {
		parent.attachEvent( "onbeforeunload", function() {
			setDocument();
		});
	}
```

parent.attachEvent being undefined in IE11.

**Comment by msevcenko** [comment]
With normal TDI reques, the node argument of the Sizzle.setDocument contains XMLDocument object, having no parentWindow, so no onbeforeunload is bound. If the TDI submits form with file data, the submit is via iframe rather than via AJAX, and the argument node for some reason contains Document object, which has a parentWindow, so IE attemts to bind and event and crashes.

The code is called from within the _success handler of TDI, namely by the code `$xml.find( 'status' );`.

**Comment by msevcenko** [comment]
Looks like this is a jQuery problem, may be related to https://bugs.jquery.com/ticket/13936, upgrading to 1.12.4 seems to fix the problem.

**Comment by centi** [comment]
Could you try version 1.10.2 as well? Does it also fix the problem?

**Comment by msevcenko** [comment]
Confirmed. Looks like the bug is fixed between 1.10.1 and 1.10.2.

**Comment by centi** [comment]
I updated the documentation, stating that the minimum required version of jQuery is now 1.10.2 and added a warning to the code as well.
